packages/pixz.rb

```diff
@@ -0,0 +1,46 @@
+# Adapted from Arch Linux pixz PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/pixz/trunk/PKGBUILD
+
+require 'package'
+
+class Pixz < Package
+  description 'Parallel, indexed xz compressor'
+  homepage 'https://github.com/vasi/pixz'
+  version '1.0.7-0829'
+  license 'BSD'
+  compatibility 'all'
+  source_url 'https://github.com/vasi/pixz.git'
+  git_hashtag '0829c7315c804a4e40abd63a9d624194dc1e4f0a'
```

Comment:
why are you using a different hashtag instead of v1.0.7?

Reply:
Great question. There are actually some bugfixes more recently which we should have, e.g.
https://github.com/vasi/pixz/commit/1aeb09b868dcac05830c513c02dcbe587d3f44e1